lms/static/sass/views/_bookmarks.scss

```diff
@@ -124,4 +125,59 @@
 
 .bookmarks-empty-detail {
   @extend %t-copy-sub1;
-}
\ No newline at end of file
+}
+
+
+// Rules for bookmark icon shown on each sequence nav item
+i.bookmarked {
+  top: -3px;
+  position: absolute;
+  left: 10px;
```

Comment:
NIT suggestion - left: 5px would give it a bit more even spacing on the left and right between the end of the element and the sequential icon.